components/cards/card.css

```diff
@@ -0,0 +1,108 @@
+@import url("../../utilities/global.css");
+@import url("../../helper/spacing.css");
+@import url("../../helper/size.css");
+@import url("../../helper/flex.css");
+@import url("../../helper/doc.css");
+
+.component-type {
+  padding-bottom: 2rem;
+}
+.component-type:not(:last-child) {
+  border-bottom: 3px solid var(--card-color);
+}
+.active-component {
+  border-radius: 0.3rem;
+  background-color: var(--card-color);
+}
+
+/* || CARD VERTICAL (1)*/
+.card {
+  display: flex;
+  flex-direction: column;
+  border-radius: 0.3rem;
+  box-shadow: 0px 0px 0.5rem 0 #e6e3e3;
+}
+.card:hover {
+  transform: scale(1.03);
+  box-shadow: 0px 0px 0.5rem 0 #c4c4c4;
+  transition: transform 0.3s;
+}
+.card-vertical {
+  max-width: 18rem;
+  margin: 0.5rem;
+}
+.card-details-container {
+  display: flex;
+  flex-direction: column;
+}
+.card-details {
+  margin-bottom: 0.5rem;
```

Comment:
Try to use var at the padding, margin, radius, etc. That will help you in future projects.

Reply:
@Raghav888 thanks for the review . im already using helper classes for few cases.. but these values are so random sometimes i dont want to declare every other value as variable..